cmake . fails

`cmake .` seems to be failing due to some missing files? I'm on `apple-arm64-darwin`. 

```
❯ cmake .
-- No build type selected, default to Release
-- Building with -DBUILD_LLVM_DWARF
-- Building with -std=c++17
-- Building with -Wall
-- Building with -Werror
-- Building with -Wextra
-- Building with -Wno-unused-parameter
-- Building with -fno-omit-frame-pointer
-- Building with -fno-rtti
-- Building with -Wno-implicit-int-float-conversion
-- Building with -Wno-unknown-warning-option
-- Building with -Wswitch
-- Building with -Wimplicit-fallthrough
-- Building with -Wnon-virtual-dtor
-- Building with -fPIC
-- Building with -O2
-- Building with -UNDEBUG
-- Building libbinaryen as shared library.
-- Configuring done
CMake Error at third_party/CMakeLists.txt:10 (add_library):
  Cannot find source file:

    googletest/googletest/src/gtest-all.cc

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .mpp .m .M .mm .ixx .cppm .h
  .hh .h++ .hm .hpp .hxx .in .txx .f .F .for .f77 .f90 .f95 .f03 .hip .ispc


CMake Error at third_party/CMakeLists.txt:14 (add_library):
  Cannot find source file:

    googletest/googletest/src/gtest_main.cc

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .mpp .m .M .mm .ixx .cppm .h
  .hh .h++ .hm .hpp .hxx .in .txx .f .F .for .f77 .f90 .f95 .f03 .hip .ispc


CMake Error at third_party/CMakeLists.txt:10 (add_library):
  No SOURCES given to target: gtest


CMake Error at third_party/CMakeLists.txt:14 (add_library):
  No SOURCES given to target: gtest_main


CMake Generate step failed.  Build files cannot be regenerated correctly.

```

--- a/README.md
+++ b/README.md
@@ -305,6 +305,15 @@ etc.
 
 ## Building
 
+Binaryen uses git submodules (at time of writing just for gtest), so before you build you will have to initialize the submodules:
+
+```
+git submodule init
+git submodule update
+```
+
+After that you can build with CMake:
+
 ```
 cmake . && make
 ```

